nifty-script

```diff
@@ -400,7 +400,9 @@ travis_for_sites() {
 
     echo "travis_fold:start:deploy"
     deploy_args=$(./prepare-deploy ${mode} ${SITE} ${TRAVIS_BUILD_NUMBER})
-    make -C ${SITE} deploy-${mode} ${deploy_args}
+    if [ $mode = "staging" ]; then
```

Comment:
OK. You're right about the tag manipulation. So this code is fine as-is. Proceed.